### PR TITLE
Add package: intltool

### PR DESCRIPTION
**Pull request by suhan-paradkar**

### CODE REVIEW

File: packages/intltool/build.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+TERMUX_PKG_VERSION=0.51.0
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_DEPENDS="perl"
+TERMUX_PKG_SRCURL=https://launchpad.net/intltool/trunk/$TERMUX_PKG_VERSION/+download/intltool-$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd
+TERMUX_PKG_PLATFORM_INDEPENDENT=true
+termux_step_create_debscripts()  {
+touch postinst
+echo "cpan install XML::Parser" >> postinst
+}
```
Comment (Grimler91) [inline packages/intltool/build.sh:14]: Have a look at some of the other debscripts, for example: https://github.com/termux/termux-packages/blob/96b4e572d692510e957b3092f3d06e77905f7128/packages/python/build.sh#L78. 

You need to add a shebang, and there's no need to touch the script. 

And please indent the function.

**Review by Grimler91 — CHANGES_REQUESTED** [review]

File: packages/intltool/build.sh (hunk 1 above)
Comment (suhan-paradkar) [reply packages/intltool/build.sh]: Added it ...

**Comment by suhan-paradkar** [comment]
@Grimler91 ,...I changed it.. is it now as per requirements..?

**Comment by Grimler91** [comment]
Yes, debscript looks good now. Just need to fix the CI so that the paclage is built correctly now

**Comment by Grimler91** [comment]
Please [rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) this against the master branch

**Comment by suhan-paradkar** [comment]
@Grimler91 ,...I changed it.. is it now as per requirements..?

> Please [rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) this against the master branch

Oh. No..! Wait did I do a mistake??

**Comment by Grimler91** [comment]
Looks like you merged instead of rebasing, so the CI will try to build all the changed packages instead of just intltool. 

To get a fresh branch with just the intltool commit now you can do

```sh
git fetch origin master
git checkout origin/master -b intltool-new
git cherry-pick 26c88a77663e291c2c0fe3a42c7007a161ee027a 18415d3ede516003d0034f81cec00f0d99de854d
git push -f name-of-your-remote intltool-new:intltool
```

where `name-of-your-remote` needs to be changed to whatever you call your remote pointing to `https://github.com/suhan-paradkar/termux-packages`.

**Comment by suhan-paradkar** [comment]
> Looks like you merged instead of rebasing, so the CI will try to build all the changed packages instead of just intltool.
> 
> To get a fresh branch with just the intltool commit now you can do
> 
> ```shell
> git fetch origin master
> git checkout origin/master -b intltool-new
> git cherry-pick 26c88a77663e291c2c0fe3a42c7007a161ee027a 18415d3ede516003d0034f81cec00f0d99de854d
> git push -f name-of-your-remote intltool-new:intltool
> ```
> 
> where `name-of-your-remote` needs to be changed to whatever you call your remote pointing to `https://github.com/suhan-paradkar/termux-packages`.

Thanks for that...

File: packages/intltool/build.sh (hunk 1 above)
Comment (Grimler91) [inline packages/intltool/build.sh:14]: The indent/shebang commit was dropped now, so this should be formatted again

Comment (suhan-paradkar) [reply packages/intltool/build.sh]: Done!

**Comment by suhan-paradkar** [comment]
@Grimler91 .. I believe this PR and mailutils are ready to be merged....

**Comment by Grimler91** [comment]
Will try it out in a fresh termux installtion to verify that the cpan postinst installation works, when I happen to be using a fresh termux installation next time

**Comment by suhan-paradkar** [comment]
> Will try it out in a fresh termux installtion to verify that the cpan postinst installation works, when I happen to be using a fresh termux installation next time

I guess you can do that with termux-docker...

**Comment by suhan-paradkar** [comment]
@Grimler91 Looks like XML::Parser requires libexpat and XML::LibXML  module as dependencies... You can see here Yisus7u7/termux-deskrop-xfce#8

**Comment by xeffyr** [comment]
So it builds a native extensions, right? In this case it should depend on `clang` and other packages necessary for installation from source.

Remove `libxml-parser-perl` - useless here since you are installing `intltool` via `cpan` in postinstall script.

P.S. The content of this pull request is subject to be moved into https://github.com/termux/unstable-packages. Packages that are involve installation of external modules are not considered to be stable. Also https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#a-note-about-package-requests:
> Packages must NOT be a part of language-specific ecosystem. These packages are installable through cpan, gem, npm, pip and similar.

Basically this package in its current implementation is nothing more that automated `cpan` wrapper.

**Comment by suhan-paradkar** [comment]
> So it builds a native extensions, right? In this case it should depend on `clang` and other packages necessary for installation from source.

I will try with revised dependencies in the build environment

> Remove `libxml-parser-perl` from setup-ubuntu script - useless here since you are installing `intltool` via `cpan` in postinstall script.

Well... I am NOT installing intltool from cpan , but only its dependencies... The package source is like any other package, just the dependencies are cpan modules

> P.S. The content of this pull request is subject to be moved into https://github.com/termux/unstable-packages. Packages that are involve installation of external modules are not considered to be stable. Also https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#a-note-about-package-requests:

Okay... I will leave that for you

**Comment by xeffyr** [comment]
https://github.com/termux/unstable-packages/pull/216
